packages/style-kit/src/_tags.scss

```
@@ -0,0 +1,43 @@
+@import './variables';
+
+@mixin tag-base($theme) {
+  @include tag-label;
+  display: inline-block;
+  padding: 6px 16px 6px 16px;
+  border-radius: $border-radius-large;
+  background: transparentize(map-get($theme, 'highlight'), 0.9);
+  color: map-get($theme, 'highlightText');
+  text-align: center;
+}
+
+@mixin edit {
+  meeco-icon,
+  svg {
+    vertical-align: middle;
+    display: inline-block;
+    padding-left: 0.5rem;
+  }
+}
+
+@mixin icon {
+  meeco-icon,
+  svg {
+    vertical-align: middle;
+    display: inline-block;
+    padding-right: 0.5rem;
+  }
+}
+
+@mixin tags-main($theme) {
+  * {
```

I think the scoping of this is a bit off - we don't want *anything* with the `.edit`/`.icon` class on it to get those styles. I think both the `.edit` and `.icon` should be nested in `.tag&`.
```suggestion
  @mixin tags-main($theme) {
     .tag {
       @include tag-base($theme);
       &.edit {
         @include edit;
       }
       &.icon {
         @include edit;
       }
     }
  }
```

Also `edit` and `icon` might want to be renamed to `tag-edit` and `tag-icon` to avoid collisions with other `edit` and `icon` mixins.